pkgs/development/libraries/cairo/default.nix

```diff
@@ -66,6 +67,7 @@ in stdenv.mkDerivation rec {
     "--enable-quartz-font"
     "--enable-quartz-image"
     "--enable-ft"
+    "--enable-xlib=no"
```

Comment:
This should not have an effect! It also shouldn't override x11Support

Reply:
Looking at it now, I’m not sure why I needed to add this change before. When I tried building cairo again from the latest Nixpkgs master (i.e., without the changes I made in this PR) it worked just fine, as you suggested. I’m sorry for the extra noise here!